### DISCUSSION TITLE
tutorials status

### DISCUSSION THREAD

**Discussion by dvirtz** [hylo-lang/hylo#806] [Q&A]
Are the tutorials expected to work with latest main?
Some of them fail to compile. For example, the `inout` snippet from https://tour.val-lang.dev/bindings fails with
```
bindings.val:14.20-21: error: type '{x: Float64, y: Float64}' has no member 'x'
  inout x = &point.x
                   ^
bindings.val:15.4-5: error: undefined name 'x' in this scope
  &x = 3.14
   ^
```
Should I open an issue?

**Comment by kyouko-taiga** [comment]
No, unfortunately not all examples in the tour are expected to work. Many features are sill under development or might be subject to change.

The tour should be understood as an glimpse of what the language will look like.
